addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -553,6 +553,56 @@ QUnit.test('chat window: open / close', async function (assert) {
     assert.verifySteps(['rpc:channel_fold/open']);
 });
 
+QUnit.test('chat window: focus on ESCAPE', async function (assert) {
+    assert.expect(1);
+
+    this.data['res.partner'].records = [{
+        email: "testpartnert@odoo.com",
+        id: 11,
+        name: "TestPartner",
+    }];
+    Object.assign(this.data.initMessaging, {
+        channel_slots: {
+            channel_channel: [
+                {
+                    channel_type: 'channel',
+                    id: 20,
+                    is_minimized: true,
+                    is_pinned: true,
+                    name: "General",
+                    state: 'open',
+                    fold_state: 'open',
+                    uuid: 'channel-20-uuid',
+                },
+                {
+                    channel_type: 'channel',
+                    id: 21,
+                    is_minimized: true,
+                    is_pinned: true,
+                    name: "other chat",
+                    state: 'open',
+                    fold_state: 'open',
+                    uuid: 'channel-21-uuid',
+                }
+            ],
+        },
+    });
+
+    await this.start({
+        hasChatWindow: true
+    });
+
+    await afterNextRender(() => {
+        const ev = new window.KeyboardEvent('keydown', { bubbles: true, key: "Escape" });
+        document.querySelector(`.o_ComposerTextInput_textarea`).dispatchEvent(ev);
+    });
+
+    assert.ok(
+        document.querySelector(`.o_ChatWindow`).classList.contains('o-focused'),
```

Comment:
You can use `assert.hasClass` for this type of assertion. Have a look in the file defining it `addons/web/static/tests/helpers/qunit_asserts.js` because there are a lot of helpful methods that you should know and use when appropriate.
(Note: we didn't use them everywhere so far and that's a mistake that we will progressively adapt. We still use too much assert.ok or assert.strictEqual)